app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -189,6 +190,12 @@ def test_str(self):
         self.assertEqual(str(branch), branch.name)
 
 
+class TestNimbusDocumentLink(TestCase):
+    def test_str(self):
+        doco_link = NimbusDocumentationLinkFactory.create()
+        self.assertEqual(str(doco_link), "%s (%s)" % (doco_link.title, doco_link.link))
```

Comment:
Yeah you can test that but it's also okay to just mark the `__str__` method as `# pragma: nocover`

Reply:
Eh, we test the name for branches, figured I might as well follow pattern

Reply:
Also format strings instead of `%` formatting here ^